x-pack/platform/plugins/shared/stack_connectors/public/connector_types/openai/constants.tsx

```diff
@@ -60,6 +61,7 @@ export const openAiConfig: ConfigFieldSchema[] = [
     label: i18n.API_URL_LABEL,
     isUrlField: true,
     defaultValue: DEFAULT_URL,
+    customValidator: validateURL,
```

Comment:
I think this field needs to be added to the `ConfigFieldSchema` interface in `x-pack/platform/plugins/shared/triggers_actions_ui/public/application/components/simple_connector_form.tsx`.

And maybe it's a nitpick, but should we call it something like `customUrlFieldValidator` or something?

```suggestion
    customUrlFieldValidator: validateURL,
```

If it is only used when `isUrlField` is `true`, then we probably should be specific.